jina/parsers/orchestrate/runtimes/worker.py

```diff
@@ -71,4 +72,32 @@ def mixin_worker_runtime_parser(parser):
 `Executor cookbook <https://docs.jina.ai/fundamentals/executor/executor-files/>`__
 ''',
     )
+
+    gp.add_argument(
```

Comment:
native is not a `workerRuntime` argument, it is a `Deployment` argument

Reply:
I will add it to `mixin_scalable_deployment_parser` so that it doesn't show up for gateway

Reply:
done